src/d/actor/d_a_mmusic.cpp

```diff
@@ -32,13 +51,22 @@ s32 daMmusic::Act_c::_create() {
 }
 
 /* 00000268-000002B4       .text _delete__Q28daMmusic5Act_cFv */
-BOOL daMmusic::Act_c::_delete() {
+bool daMmusic::Act_c::_delete() {
     /* Nonmatching */
+    int iVar1;
+    if (this->field_0x2A0 != 0) {
```

Comment:
Looking a bit more at the actor, _execute sets this field to a JPABaseEmitter*, so I'm guessing this is:

```cpp
bool daMmusic::Act_c::_delete() {
    if (mEmitter != NULL) {
        mEmitter->becomeInvalidEmitter();
        mEmitter = NULL;
    }
    delete_se();
    return true;
}
```

which is a pattern that shows up elsewhere.